### ISSUE TITLE
Crash when loading Schematic

### ISSUE THREAD

**Issue by Joshi120** [Lunatrius/Schematica#174]
Hello each time i try to load a Schematic i crash after a few secconds MC version 1.10.2

```
java.lang.ClassCastException: com.github.lunatrius.schematica.client.world.chunk.ChunkProviderSchematic cannot be cast to net.minecraft.client.multiplayer.ChunkProviderClient
	at net.minecraft.client.multiplayer.WorldClient.func_72863_F(WorldClient.java:508)
	at net.minecraft.client.multiplayer.WorldClient.func_72863_F(WorldClient.java:49)
	at net.malisis.core.util.chunkblock.ChunkBlockHandler.getAffectedChunks(ChunkBlockHandler.java:338)
	at net.malisis.core.util.chunkcollision.ChunkCollision.getRayTraceResult(ChunkCollision.java:132)
	at net.minecraft.world.World.func_147447_a(World.java:903)
	at com.github.lunatrius.schematica.handler.client.RenderTickHandler.rayTrace(RenderTickHandler.java:50)
	at com.github.lunatrius.schematica.handler.client.RenderTickHandler.onRenderTick(RenderTickHandler.java:23)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_1140_RenderTickHandler_onRenderTick_RenderTickEvent.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:72)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:168)
	at net.minecraftforge.fml.common.FMLCommonHandler.onRenderTickStart(FMLCommonHandler.java:350)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1074)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:371)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
```

**Comment by RedNesto** [comment]
Are you using the latest version of Schematica ?
Because this was fixed by https://github.com/Lunatrius/Schematica/commit/51d68b5a0270b7356dea7818764a587b77671621

**Comment by Joshi120** [comment]
o wow yes i had a old version so nevermind then ...

**Comment by Lunatrius** [comment]
Yeah, was fixed, also a duplicate of #168.
